packages/webpack/src/config/server.js

```diff
@@ -61,6 +61,27 @@ export default class WebpackServerConfig extends WebpackBaseConfig {
     return plugins
   }
 
+  extendConfig() {
+    const config = super.extendConfig(...arguments)
+
+    // Add minimizer plugins
+    if (config.optimization.minimize && config.optimization.minimizer === undefined) {
```

Comment:
Since both client and server need minimize, can we move it into base ?

Reply:
There may be more differences such as the way of comment extraction. What do you think to add `minimizer()` function into base and override it in server?

Reply:
Yes, `minimizer` is a good idea